spec/openapi.yaml

```diff
@@ -1138,6 +1157,46 @@ components:
                 description: Path to package
                 example: "jo/packages"
 
+    Job:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 42
+        name:
+          type: string
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        path:
+          type: string
+          example: "/local/Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        display_path:
+          type: string
+          example: "/PrusaLink gcodes/Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        size:
+          type: integer
+          example: 23931
+        filament_type:
+          type: string
+          example: "PLA"
+        print_height:
+          type: number
+          example: 0.3
+        m_timestamp:
+          type: integer
+          example: 1648042843
+        state:
+          type: string
+          enum: [ PRINTING, PAUSED, FIN_STOPPED, FIN_ERROR, FIN_OK, FIN_HARVESTED ]
```

Comment:
Are we sure these states are a reasonable set? What about some kind of ERROR / ATTENTION /? How does _this_ state and the printer state relate to each other?

Reply:
My bad, I think there should really be the printer state, I'll fix it